pangocairo/src/font_map.rs

```diff
@@ -30,7 +30,7 @@ impl FontMap {
     }
```

Comment:
^------ this doesn't return an `Option`

Reply:
The new function is not nullable, I mean

Reply:
Oh right!